data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+				
+			`	"Interesting finding, Captain. Is there anything else that he discovered?"`
+			choice
+				`	"No."`
+				`	"Yes. We used gravitational mapping equipment on Midgard and we found what looked like an artificial singularity."`
+					goto singularity
+				
+			apply
+				set "playing coy"
+			`	The woman does not respond for a few seconds. When she does begin speaking, you have to lean in to hear her. "Don't lie to me. I can make this experience very uncomfortable for you."`
+			choice
+				`	"Alright. We discovered an artificial singularity on Midgard."`
+			
+			label singularity
+			branch honest
+				not "playing coy"
+
+			apply
+				clear "playing coy"
+			`	"See? That wasn't so hard to admit, now was it?"`
+			
+			label honest
+			branch helped
+				has "Deep: Mystery Cubes 4: done"
+			
+			`	The woman's mouth turns into a sly smile as she leans back in her chair. "Thank you for the information. That is all we needed you for. You will be escorted back outside momentarily. Enjoy the rest of your day, <first> <last>."`
+				goto next
+			
+			label helped
+			apply
+				set "deep: helped before archaeology"
+			`	The woman's mouth turns into a sly smile as she leans back in her chair. "Thank you for the information, Captain. Before I let you go, I'd like to thank you for the work you've done for the Deep. I was interested to hear that the same Captain who helped us pursue Beelzebub also helped Albert Foster. Don't worry, we all make stupid mistakes. We'll scratch this one off your record.`
+			`	"You clearly mean no harm to the Deep, so I'll give you the privilege of asking me any questions you might have after this encounter."`
+			choice
+				`	"When was the Deep really settled?"`
+				`	"Is there really an artificial singularity on Midgard?"`
+					goto singularity?
+				`	"I don't have any questions."`
+					goto none
+				
+			`	"The exact date has been lost to time, but folklore states that the first humans arrived here in the mid-1300s, before the time of the 'Black Death' plague on ancient Earth."`
+			choice
+				`	"Were the original humans of the Deep really brought here by aliens?"`
+				`	"I don't have any more questions."`
+					goto none
+				
+			`	"That's a question for the people who were originally brought here." The woman flashes another sly smile.`
+			choice
+				`	"Is there really an artificial singularity on Midgard?"`
+				`	"I don't have any more questions."`
+					goto none
+				
+			`	"Those gravitational mapping devices can be so finicky sometimes. I'm sure it was nothing."`
+				goto done
+			
+			label singularity?
+			`	"Those gravitational mapping devices can be so finicky sometimes. I'm sure it was nothing."`
+			choice
+				`	"When was the Deep really settled?"`
+				`	"I don't have any questions."`
+					goto none
+				
+			`	"The exact date has been lost to time, but folklore states that the first humans arrived here in the mid-1300s before the time of the 'Black Death' plague on ancient Earth."`
```

Comment:
Add a comma after "mid-1300s" to match the otherwise identical text above.

Reply:
Lines 990 and 1012 are also missing the comma.

Reply:
(I recommend "Start a review", you'll be able to provide a summary statement, in addition to your line-specific comments.)

Reply:
Thanks. I think I'll actually fork this repo and make a PR there instead.

Reply:
>Thanks. I think I'll actually fork the origin repo and make a PR there instead.

I probably won't be accepting all these changes, and I'll have time tomorrow morning to tackle this anyway, so a PR isn't necessary.

Reply:
Okay. Starting review then.